scripts/eosio_build_amazon.sh

```diff
@@ -115,7 +115,7 @@ fi
 printf "\\n"
 
 ### clean up force build before starting
-if [ $FORCE_BUILD ];then
+if $FORCE_BUILD;then
```

Comment:
I thought that only worked if `$FORCE_BUILD` was `true` or `false` not 1 or 0 which `eosio_build.sh` uses. Should we not use `[]` for most portable use?

Reply:
I got confused since I moved all of the new scripts I've been writing to use booleans. This actually exposes the reason I dislike using integers: [ $FORCE_BUILD ] if FORCE_BUILD=0 or 1, or even true or false, will ALWAYS return true:
```
 nathan.pierce@npierce-mbp  ~/BLOCKONE/eos   minor-fixes  bash -c "TEST=0; if [ \$TEST ]; then echo 'GOT IT'; fi"
GOT IT
 nathan.pierce@npierce-mbp  ~/BLOCKONE/eos   minor-fixes  bash -c "TEST=1; if [ \$TEST ]; then echo 'GOT IT'; fi"
GOT IT
 nathan.pierce@npierce-mbp  ~/BLOCKONE/eos   minor-fixes  bash -c "TEST=true; if [ \$TEST ]; then echo 'GOT IT'; fi"
GOT IT
 nathan.pierce@npierce-mbp  ~/BLOCKONE/eos   minor-fixes  bash -c "TEST=false; if [ \$TEST ]; then echo 'GOT IT'; fi"
GOT IT
 nathan.pierce@npierce-mbp  ~/BLOCKONE/eos   minor-fixes  bash -c "TEST=; if [ \$TEST ]; then echo 'GOT IT'; fi"
 nathan.pierce@npierce-mbp  ~/BLOCKONE/eos   minor-fixes                                                  
```
Fortunately, if it's not set (which it doesn't get set by default right now), we'll be ok.

Reply:
Sorry about that. Reverting now.